packages/lodestar/src/chain/emitter.ts

```diff
@@ -40,6 +39,8 @@ export interface IChainEvents {
   "forkChoice:reorg": (head: IBlockSummary, oldHead: IBlockSummary, depth: number) => void;
   "forkChoice:justified": (checkpoint: Checkpoint) => void;
   "forkChoice:finalized": (checkpoint: Checkpoint) => void;
+
+  "block:unknownRoot": (root: Root) => void;
```

Comment:
isn't that present in error:block?

Reply:
when doing block `validate` we throw `error:block` with `ERR_PARENT_UNKNOWN` code and chain handle that, then chain puts to pending block pool and emits `block:unknownRoot` with missing ancestor root then sync will handle that.
So I'd separate `error:block` and `block:unknownRoot` event.

Reply:
hmm but do we need to fire block:unknownRoot from pool again, I mean everyone can listen to error:block and check if code is missing parent, right?

Reply:
I thought about it, my understanding is `BlockPool` merely a data storage, chain is the controller so I did not fire `block:unknownRoot` inside block pool and let chain does the job.
cc @wemeetagain to see if this is the correct way